.github/workflows/main.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on: [ push, pull_request ]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        platform: [ ubuntu-latest ]
+        node: [ '8', '10', '12', '14' ]
+    name: Node ${{ matrix.node }} (${{ matrix.platform }})
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+      - name: install dependencies
+        run: npm ci
+      - name: Lint
+        run: npm run lint
+      - name: Unit Tests
+        run: npm run test:coverage
+      - name: Prepare Coveralls
+        run: npm run coveralls
```

Comment:
I think you run tests twice,
`npm run coveralls` runs `npm run test:coverage` which means you run tests twice, no?

Reply:
Right, removed duplicate tests running